en/docs/learn/kerberos-grant.md

```diff
@@ -74,15 +74,15 @@ Kerberos ticket for an OAuth2 token.
 
 Follow the instructions below to configure Kerberos Grant with WSO2 IS:
 
-1.  Download the [kerberos grant.jar](../assets/attachments/org.wso2.carbon.identity.sample.extension.authenticators-5.7.0.jar) file.
+1.  Download the [kerberos grant.jar](../assets/attachments/kerberos-grant-1.0.0.jar) file.
 2.  Copy the JAR into the `<IS_HOME>/repository/components/lib` directory.
-3.  Add the following entry to the `deployment.toml` file in the `<IS_HOME>/repository/conf/` folder.
+3.  Add the following entry to the `deployment.toml` file in the
+    `<IS_HOME>/repository/conf/` folder to enable **Kerberos** grant
+    type.
```

Comment:
To enable the **Kerberos** grant, add the following entry to the `deployment.toml` file in the `<IS_HOME>/repository/conf/` folder.